src/components/Box/Box.js

```diff
@@ -3,11 +3,12 @@ import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { space, color } from 'styled-system';
```

Comment:
should we eventually change this to use the system props exported from ../../utils/constants too?
e.g. layout, space, flex, and color would become COMMON, FLEX, COLOR?